src/particles/wakefields/WakePush.H

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2022-2023 The Regents of the University of California, through Lawrence
+ *           Berkeley National Laboratory (subject to receipt of any required
+ *           approvals from the U.S. Dept. of Energy). All rights reserved.
+ *
+ * This file is part of ImpactX.
+ *
+ * Authors: Alex Bojanich, Chad Mitchell, Axel Huebl
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef WAKEPUSH_H
+#define WAKEPUSH_H
+
+#include <AMReX_REAL.H>
+#include <vector>
+#include "../ImpactXParticleContainer.H"
+
+namespace impactx::wakepush
```

Comment:
mirror the names of the directories this file is in:
```suggestion
namespace impactx::particles::wakefields
```